### PR TITLE
javascript

### PR DESCRIPTION
**Pull request by jiu33**

### CODE REVIEW

**Comment by whhe** [comment]
Hi @jiu33, thanks for your contribution! I just updated the CI workflow configurations on master branch, could you please rebase on it and push your branch again? You also need to add a workflow yaml file  `javascript.yml` like below:

```yaml
name: JavaScript CI

on:
  push:
    paths:
      - '.github/workflows/javascript.yml'
      - 'javascript/**'
  pull_request:
    paths:
      - '.github/workflows/javascript.yml'
      - 'javascript/**'

jobs:
  ci:
    strategy:
      matrix:
        module:
          - name: 'mysql2'
            with_oceanbase_container: true
    uses: ./.github/workflows/basic-workflow.yml
    with:
      language: 'javascript'
      module: ${{ matrix.module.name }}
      with_oceanbase_container: ${{ matrix.module.with_oceanbase_container }}
```

File: javascript/mysql2/README-CN.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+npm init -y
+
+npm install mysql2
+
+```
+
+创建 [example.js](example.js) 文件
+
+```
+onst mysql = require('mysql2');
```
Comment (whhe) [inline javascript/mysql2/README-CN.md:25]: Typo: 'onst' -> 'const'

File: javascript/mysql2/README.md [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+npm init -y
+
+npm install mysql2
+
+```
+
+创建 [example.js](example.js) 文件
+
+```
+onst mysql = require('mysql2');
```
Comment (whhe) [inline javascript/mysql2/README.md:25]: const

File: javascript/mysql2/README.md [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+```
+mkdir example
+cd example
+npm init -y
+
+npm install mysql2
+
+```
+
+创建 [example.js](example.js) 文件
```
Comment (whhe) [inline javascript/mysql2/README.md:22]: This should be in English right?

**Review by whhe — COMMENTED** [review]
Thanks for your contribution! I left some comments.

File: javascript/mysql2/README.md (hunk 3 above)
Comment (jiu33) [reply javascript/mysql2/README.md]: sorry I do it

**Review by whhe — APPROVED** [review]
LGTM

**Pull request merged**